tests/test_jmcomic/test_jm_async_custom.py

```
@@ -0,0 +1,97 @@
+            client_key = 'my_async_test'
+
+        JmModuleConfig.register_async_client(MyAsyncClient)
+
+        # 通过 option 创建自定义 client
+        opt = self.new_option()
+        opt.client.src_dict['async_impl'] = 'my_async_test'
+        loop = asyncio.new_event_loop()
+        client = loop.run_until_complete(opt.new_jm_async_client())
+
+        self.assertIsInstance(client, MyAsyncClient)
+        # 域名应回退到默认 API 域名列表（与 sync 行为一致）
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)